src/parsers/power_system_table_data.jl

```diff
@@ -216,8 +216,12 @@ end
 
 """Return the dataframe for the category."""
 function get_dataframe(data::PowerSystemTableData, category::InputCategory)
-    @assert haskey(data.category_to_df, category)
-    return data.category_to_df[category]
+    if !haskey(data.category_to_df, category)
+        @warn("Missing $category data.")
+        return nothing
```

Comment:
This implies that every call site needs to check whether the returned value is `nothing`.  In what scenario will there be a missing category?

Reply:
I have an example where the tabular data file doesn't have a reserves.csv file. It's perfectly valid to not have reserves, and therefore not create `service` components. It wasn't an obvious problem to debug when I encountered it... in fact, it wasn't really a problem. But I do want to be notified about not having information that might be expected.

Reply:
Could we create an empty dataframe instead of nothing?